client-tmp/react/README.md

```diff
@@ -0,0 +1,88 @@
+
+# GenesisLCAP React Showcase Client Application
```

Comment:
Copied from integration repo I'm guessing :) Have a look at the Web components readme

Reply:
I added the basic information + npm scripts to be consistent with those from the web components.